### ISSUE TITLE
genomeAssembly

### ISSUE THREAD

**Issue by pnrobinson** [phenopackets/phenopacket-schema#93]
@julesjacobsen 
The VcfAllele needs a genomeAssembly. I am not that familiar with SPDI but probably it wouldnt harm to add one there as well. I think we should not add the genomeAssembly to the variant itself, because it is not directly relevant for HGVS, ICSN, or MouseAllele. 

We could use either a string or an enumeration. I would favor a string because other than "UNKNOWN" there is no truly natural default genome build. And if we want the schema to be species neutral, then we would have a difficult maintainence task (as well as a very long enumeration) if we wanted to list all species.

So basically we can leave it to the validation code to check for the existance of a genome Assembly, and it we are doing a specifici validation (e.g., human rare disease) then we can check for specific genome builds.

**Comment by julesjacobsen** [comment]
Agreed. SPDI doesn't require an assembly as this is implicitly specified by the sequence identifier, like HGVS. Likewise I agree that, unfortunately a string is the only workable datatype for this.

This should probably be changed in the ```HtsFile``` message too.
